_inc/client/plans/product-selector.jsx

```diff
@@ -46,6 +46,14 @@ class ProductSelector extends Component {
 			},
 		} );
 
+		const renewalDate = __( 'Renews on %(renewalDate)s', {
+			args: {
+				renewalDate: moment( purchase.expiry_date ).format( 'LL' ),
+			},
+		} );
+
+		const subtitleText = purchase.is_refundable ? purchasedDate : renewalDate;
```

Comment:
Question on the logic here: should we display "Renews on" only when `is_renewable` is `true`?

Reply:
From my understanding, the original intent was to show the purchase date while the refund window was open (which I assume means `is_renewable === true`), and then show the renewal date, which is the logic above. See p8oabR-qW-p2#comment-3700 where I am getting that from :)